docs/install-filecoin/install.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Install
+description: Set up the Lotus Filecoin client on your computer, ready to be used as a single node or in a mining operation.
+---
+
+# Install
+
+Install the Lotus Filecoin client on your system, ready to be used as a single node or in a mining operation. To make the install go as smoothly as possible, read through the [software and hardware requirements](/install-filecoin/requirements) first.
```

Comment:
```suggestion
Install the Lotus Filecoin client on your system, ready to be used as a single node or in a mining operation. To make the install go as smoothly as possible, read through the [software and hardware requirements](./requirements.md) first.
```

For internal documentation links use relative paths to the files then you get two benefits:
1) You can browse it on Github and the links work
2) The app will notify you when you move content and break a link creating a 404